app/controllers/films_controller.rb

```diff
@@ -0,0 +1,16 @@
+class FilmsController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:index, :show]
+  def index
+    @films = Film.includes(:ratings).all
+    @film.to_json({methods: :average_rating})
+  end
+
+  def show
+    @film = Film.find_by(id: params[:id])
```

Comment:
If you let this error with find rather than find_by you'll get a 404 in production